go.mod

```diff
@@ -61,7 +61,7 @@ require (
 	google.golang.org/grpc v1.57.0
 )
 
-require github.com/vektra/mockery/v2 v2.32.3
+require github.com/vektra/mockery/v2 v2.32.4
```

Comment:
Should we also update the below?:
* https://github.com/Finschia/ostracon/blob/main/.github/workflows/check-generated.yml#L29C28-L29C34

Reply:
Thank you for bringing this to our attention. This PR has no fails running check-generated.yml. However, I think the versions should be matched if possible, so I fixed and pushed it again.